server/routes/setup_terraform_environment.py

```
@@ -184,17 +147,10 @@ def setup_aws_terraform_environment_isolated(user_id: str):
-            result = setup_aws_credentials_cached(user_id, return_isolated=True)
-            if len(result) == 4:  # New format with isolated env
-                ok_cached, cached_region, auth_method, isolated_env = result
-                if ok_cached and cached_region and isolated_env:
-                    logger.info(f"(cached) AWS credentials configured for Terraform in region: {cached_region} (isolated)")
-                    return True, cached_region, isolated_env
-            else:  # Fallback to old format
-                ok_cached, cached_region, _ = result
-                if ok_cached and cached_region:
-                    logger.info(f"(cached) AWS credentials configured for Terraform in region: {cached_region}")
-                    return True, cached_region, None
+            ok_cached, cached_region, auth_method, isolated_env = setup_aws_credentials_cached(user_id)
```

Ruff (0.15.10)

[warning] Unpacked variable `auth_method` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)